### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ruchida**
Added SugarCAL under the Calendaring category.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ This is your repository! Everyone is welcome to be involved by creating listings
 ## Business Intelligence
 
 ## Calendaring
-
+* [SugarCAL - Real-time sync between SugarCE Calendar and Google](https://sourceforge.net/projects/sugarcal/) on Sourceforge.net. ) on Sourceforge.net.
```
Comment (mikhailkozlov) [inline README.md]: Please remove last ) and add new line after your listing. Thanks.

File: README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ This is your repository! Everyone is welcome to be involved by creating listings
 ## Business Intelligence
 
 ## Calendaring
-
+* [SugarCAL - Real-time sync between SugarCE Calendar and Google](https://sourceforge.net/projects/sugarcal/) on Sourceforge.net.
 ## Campaign Management
```
Comment (mikhailkozlov) [inline README.md]: Apologies, looks like you need another new line here. Next category runs over http://cl.ly/image/2b2F2u1p2H3u.

You can use preview in github before submitting PR.

Comment (mikhailkozlov) [inline README.md]: nvm, I got

**Comment by ruchida** [comment]
Thanks!!

**Pull request merged**